packages/xcm-util/convert-xcm-version/upgrade-xcm-version.ts

```diff
@@ -221,6 +293,28 @@ function upgradeNetworkIdV2(networkId: NetworkIdV2): NetworkIdV3 | null {
   }
 }
 
+function upgradeNetworkIdV4(networkId: NetworkIdV3): NetworkIdV5 {
+  if (
+    networkId === 'westend' ||
+    networkId === 'rococo' ||
+    networkId === 'wococo'
+  ) {
+    throw new Error(`upgradeNetworkIdV4: ${networkId} is not supported in V5`);
```

Comment:
It can be upgraded using `ByGenesis`.

- Rococo Genesis Hash: `0x6408de7737c59c238890533af25896a2c20608d8b380bb01029acb392781063e`
- Westend Genesis Hash: `0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e`
- Wococo Genesis Hash: a dummy hash is used for "defunct networks" such as Wococo (all-zero 32-byte hash). After some time, Rococo will also transition to this.